packages/d3fc-chart/src/cartesian.js

```diff
@@ -76,12 +77,25 @@ export default (...args) => {
                     const canvas = select(nodes[i])
                         .select('canvas')
                         .node();
-                    webglPlotArea.context(canvas.getContext('webgl'))
+                    webglPlotArea.context(isContextLost ? null : canvas.getContext('webgl'))
                         .xScale(xScale)
                         .yScale(yScale);
                     webglPlotArea(d);
                 });
 
+            container.select('.webgl-plot-area canvas')
```

Comment:
`.webgl-plot-area>canvas`

I'm only being fussy because of the fix for the next comment being required...